src/components/common/form/InputFile.web.js

```diff
@@ -56,91 +21,71 @@ const toBase64 = file =>
   })
 
 const InputFile = props => {
+  const { children, onChange } = props
   const inputRef = useRef(null)
 
-  const getReducedFileAsDataUrl = async (file, maxWidth = MAX_WIDTH, maxHeight = MAX_HEIGHT) => {
+  const getReducedFileAsDataUrl = useCallback(async (file, maxWidth = MAX_WIDTH, maxHeight = MAX_HEIGHT) => {
     const data64 = await toBase64(file)
     const img = await getImageWithSrc(data64)
-    const orientation = await getOrientation(file)
 
-    const dataUrl = getReducedDataUrlWithImage(img, orientation, maxWidth, maxHeight)
-    getOrientation(file, orientation => alert(orientation))
+    // standard orientation
+    const orientation = 1
```

Comment:
where 1 = portrait comes from ? where this constant is defined ? import the constant from the library or define the constant if getReducedDataUrlFromImage is written by us